monitoring/git_server.go

```diff
@@ -83,6 +83,7 @@ func GitServer() *Container {
 							PanelOptions:    PanelOptions().LegendFormat("running commands").Unit(Seconds),
 							Owner:           ObservableOwnerCloud,
 							PossibleSolutions: `
+								- **Query a graph for indivual commands** using "sum by (cmd)(src_gitserver_exec_running)" to gain more insights.
```

Comment:
```suggestion
								- **Query a graph for indivual commands** using 'sum by (cmd)(src_gitserver_exec_running)' in Grafana ('/-/debug/grafana') to see if a command might be spiking in frequency.
```

(single-quotes get replaced with markdown code backticks 🚀 )

Reply:
>(single-quotes get replaced with markdown code backticks 🚀 )

TIL :)